client-common/src/main/java/org/apache/cassandra/sidecar/common/data/RestoreJobConstants.java

```diff
@@ -36,6 +36,17 @@ public class RestoreJobConstants
     public static final String JOB_CONSISTENCY_LEVEL = "consistencyLevel";
     public static final String JOB_OPERATION_REASON = "reason";
     public static final String JOB_LOCAL_DATA_CENTER = "localDatacenter";
+    /**
+     * A boolean field; when restoreToLocalDatacenterOnly is set to true, localDatacenter must also be specified, and
+     * the restore job will only restore to the specified localDatacenter, regardless of the replication configuration
+     * of the belonging keyspace.
+     * The option should be used with caution. To help build an educated decision, when only restoring to the local
+     * datacenter, but the table replicates to multiple datacenters, it could cause a large amount of repair streaming
+     * traffic. Argurably, you might want to leverage the intra-node repair feature, then you can enable this option.
+     * Another use case that could justify the option is runing distinct restore jobs, one per datacenter, concurrently.
+     * In this case, there is an external coordinator that manges the resotre job in each datacenter.
```

Comment:
minor typos
```suggestion
     * traffic. Arguably, you might want to leverage the intra-node repair feature, then you can enable this option.
     * Another use case that could justify the option is running distinct restore jobs, one per datacenter, concurrently.
     * In this case, there is an external coordinator that manges the restore job in each datacenter.
```

Reply:
ah. good catch! Somehow, typo check is disabled in my local. :|